buildscripts/build-native.sh

```diff
@@ -16,6 +16,11 @@ check_native_prereqs()
 
     # Check for clang
     hash clang-$__ClangMajorVersion.$__ClangMinorVersion 2>/dev/null ||  hash clang$__ClangMajorVersion$__ClangMinorVersion 2>/dev/null ||  hash clang 2>/dev/null || { echo >&2 "Please install clang before running this script"; exit 1; }
+
+    # Check for additional prereqs for wasm build
+    if [ $__BuildArch == "wasm" ]; then
+        hash emcmake 2>/dev/null || { echo >&2 "Please install Emscripten before running this script"; exit 1; }
```

Comment:
It might be good to put in a pointer to https://github.com/dotnet/corert/blob/master/Documentation/how-to-build-WebAssembly.md or https://kripken.github.io/emscripten-site/docs/getting_started/downloads.html

Reply:
Will do. Will also need to check for the `$EMSCRIPTEN` variable and advise the user to run `source ./emsdk_env.sh`